app/controllers/issue_comments_controller.rb

```diff
@@ -59,11 +59,11 @@ def send_notifications
     if @project.moderator?(current_account) && visible_to_reporter?
       email = @issue.reporter.email
       commenter_kind = "moderator"
-      NotificationService.notify(account: @issue.reporter, project: @project, issue_id: @issue.id, issue_comment_id: @comment.id)
+      Resque.enqueue(NotificationWorker, @issue.reporter.id, @project.id, @issue.id, @comment.id)
```

Comment:
What do you think about moving the Rescue.enqueue call to the NotificationService class?

Reply:
Thanks!